src/EditorFeatures/VisualBasic/ExtractInterface/ExtractInterfaceCommandHandler.vb

```diff
@@ -12,5 +13,9 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.ExtractInterface
     Friend Class ExtractInterfaceCommandHandler
         Inherits AbstractExtractInterfaceCommandHandler
 
+        <ImportingConstructor>
+        Public Sub New(threadingContext As IThreadingContext)
+            MyBase.New(threadingContext)
+        End Sub
```

Comment:
weird that we need a VB specific version of this... why is this not just a single command handler defined for C# and VB content types?

Reply:
/shrug  

Idk. That's the current implementation. Is there a strong reason to change it? Maybe worthy of just a code improvement issue or something.

Reply:
/shrug is fine with me :)   just something i noticed.